AppCenterCrashes/AppCenterCrashesTests/MSCrashesTests.mm

```diff
@@ -51,11 +57,23 @@ @implementation MSCrashesTests
 - (void)setUp {
   [super setUp];
   self.sut = [MSCrashes new];
+  [MSDeviceTracker resetSharedInstance];
+  self.deviceTrackerMock = OCMClassMock([MSDeviceTracker class]);
+  OCMStub(ClassMethod([self.deviceTrackerMock sharedInstance])).andReturn(self.deviceTrackerMock);
```

Comment:
`sharedInstance` only available for class method so `ClassMethod` could be omitted.